crates/astria-sequencer-relayer/tests/blackbox/helper.rs

```diff
@@ -180,7 +190,18 @@ pub struct TestSequencerRelayer {
 
 impl Drop for TestSequencerRelayer {
     fn drop(&mut self) {
-        self.sequencer_relayer.abort();
+        // We drop the shutdown controller here to cause the sequencer relayer to shut down.
+        let _ = self.relayer_shutdown_controller.take();
+
+        let sequencer_relayer = mem::replace(&mut self.sequencer_relayer, tokio::spawn(async {}));
+        let _ = futures::executor::block_on(async move {
+            timeout(Duration::from_secs(30), sequencer_relayer)
+                .await
+                .unwrap_or_else(|_| {
+                    panic!("timed out waiting for sequencer relayer to shut down");
```

Comment:
I generally avoid putting panicking code in drop impls, but I think it's probably ok here since this is a test-only object, and having a test process abort seems like a better option than just logging here since the log message could easily be missed.